ffc/codegeneration/ufc.h

```diff
@@ -413,7 +413,8 @@ extern "C"
       const ufc_scalar_t* c,
       const double* restrict coordinate_dofs,
       const int* entity_local_index,
-      const int* cell_orientation);
+      const int* cell_orientation,
+      const int* quadrature_permutation);
```

Comment:
Please add a thorough documentation into the docstring above. The docstring above is important for anyone willing to comply with FEniCS interface.

Reply:
Done. I think cell_orientation is no longer used anywhere, but I need to check this and if so will remove it in a separate PR.